api.md

```diff
@@ -322,9 +322,9 @@ Outputs information in the following form:
 
 ⚠ This could break your feed. Please don't run this unless you understand it.
 
-Delete a message by its message key. This only deletes the message from your
-local database, not the network, and could have unintended consequences if you
-try to delete a single message in the middle of a feed.
+Delete a message by its message or feed key. This only deletes the message from
```

Comment:
This should probably be clarified to be something like: 

Delete a message by its message or feed key

=>

Delete a message by its message key or a whole feed by its key.

Reply:
Good idea, thanks!